microprofile/fault-tolerance/src/test/java/io/helidon/microprofile/faulttolerance/FaultToleranceTest.java

```diff
@@ -77,7 +77,7 @@ protected static <T> T newNamedBean(Class<T> beanClass) {
 
     static void printStatus(String message, String status) {
         System.out.println(message + " -> " + status + " [Thread: "
```

Comment:
In general printing anything to System.out in tests is not a good option, would be better to validate using an assertion (if needed). Otherwise this produces output that is not necessary

Reply:
Yeah, those have been there for a while. I will switch them to logs.